README.md

```diff
@@ -541,6 +541,7 @@ Table of Contents
    * [inspectlet.com](http://inspectlet.com/) — 100 sessions/month free for 1 website
    * [mousestats.com](https://mousestats.com/) — 100 sessions/month free for 1 website
    * [metrica.yandex.com](https://metrica.yandex.com/) — Unlimited free analytics
+   * [getjaco.com](http://www.getjaco.com/) — User session recording for websites
```

Comment:
I don't see the free edition listed in the pricing page - only a trial - got a link?